runtime/doc/lsp.txt

```diff
@@ -1444,6 +1444,9 @@ format({opts})                                          *vim.lsp.buf.format()*
                   using (1,0) indexing. Can also be a list of tables that
                   contain `start` and `end` keys as described above, in which
                   case `textDocument/rangesFormatting` support is required.
+                • {notify}? (`boolean`, default: true) Notify if there are
```

Comment:
the usual name is `silent`. look at other parts of this doc for examples and use the same conventions.